scala/spark-wordcount/README.md

```diff
@@ -38,10 +52,39 @@ Create a Cloud DataProc instance:
 
 ## Build the jar
 
-The Spark job is assembled into a fat jar with all of its dependencies. To build, run:
+The Spark application is assembled into a fat jar with all of its dependencies. To build, run:
 
     mvn assembly:assembly
 
+## Test your job locally with Cloud Bigtable Emulator (optional but recommended)
```

Comment:
Shouldnt this come after the environment variables section? And it should also use those variables instead of creating new ones

Reply:
Since we need to `$(gcloud beta emulators bigtable env-init)` to initialize the env these would interfere and may cause issues (unless you explicitly say that you should re-set the envs). It'd be too much for people new to Spark, HBase, Bigtable, Scala and so one IMHO.

Reply:
Then maybe merge the environment section into the others. Right now the flow of the document is quite odd.

Maybe something like this:

```
## Running locally with the Bigtable Emulator
PROJECT_ID=fake-project
INSTANCE_ID=fake-instance

gcloud beta emulators bigtable start &
$(gcloud beta emulators bigtable env-init)

spark-submit ....$PROJECT_ID $INSTANCE_ID

## Running locally using Cloud Bigtable
unset BIGTABLE_EMULATOR_HOST
PROJECT_ID=[your google cloud project id]
INSTANCE_ID=[your cloud bigtable instance id]

spark-submit ... $PROJECT_ID $INSTANCE_ID

## Running on Dataproc with Cloud Bigtable

unset BIGTABLE_EMULATOR_HOST
PROJECT_ID=[your google cloud project id]
INSTANCE_ID=[your cloud bigtable instance id]
SPARK_CLUSTER=[your dataproc cluster name]

...
```

Reply:
fixed